presidio-analyzer/analyzer/analyzer_engine.py

```diff
@@ -118,6 +147,9 @@ def analyze(self, text, entities, language, all_fields):
             # analyze using the current recognizer and append the results
             current_results = recognizer.analyze(text, entities, nlp_artifacts)
             if current_results:
+                for cur_res in current_results:
```

Comment:
I'm not sure I understand this part. Are we putting the recognizer name as a part of the result description? Can it be done when the RecognizerResult is created inside a recognizer?